src/elements/content-sidebar/MetadataInstanceEditor.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+    MetadataInstanceForm,
+    type MetadataTemplateInstance,
+    UnsavedChangesModal,
+    type MetadataTemplate,
+} from '@box/metadata-editor';
+import React from 'react';
+
+export interface MetadataInstanceEditorProps {
+    isBoxAiSuggestionsEnabled: boolean;
+    isUnsavedChangesModalOpen: boolean;
+    template: MetadataTemplateInstance | MetadataTemplate;
+}
+
+const MetadataInstanceEditor: React.FC<MetadataInstanceEditorProps> = ({
+    isBoxAiSuggestionsEnabled,
+    isUnsavedChangesModalOpen,
+    template,
+}) => {
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    const handleSubmit = () => {}; // TODO in a future PR
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    const handleCancel = () => {}; // TODO in a future PR
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    const handleDelete = () => {}; // TODO in a future PR
+
+    return (
+        <>
+            <MetadataInstanceForm
+                isAiSuggestionsFeatureEnabled={isBoxAiSuggestionsEnabled}
+                isLoading={false}
```

Comment:
should it be dynamic?

Reply:
Yes, probably, we discussed that and it looks like a bigger topic. I created a ticket for the next sprint to look into it more across different components.